libs/community/langchain_community/vectorstores/duckdb.py

```diff
@@ -72,14 +79,14 @@ class DuckDB(VectorStore):
     """
 
     def __init__(
-        self,
-        *,
-        connection: Optional[Any] = None,
-        embedding: Embeddings,
-        vector_key: str = "embedding",
-        id_key: str = "id",
-        text_key: str = "text",
-        table_name: str = "vectorstore",
+            self,
```

Comment:
I'm not sure about this new ident formatting. Did you run [`make format`](https://python.langchain.com/docs/contributing/code/#formatting-and-linting) for changes?

Reply:
I executed `make format` and it made 423 changes in many files :-)
I committed only the change in duckdb.py and rolled back the rest.